scenes/globals/pause/pause_overlay.gd

```diff
@@ -3,10 +3,12 @@
 extends CanvasLayer
 
 const TITLE_SCENE: PackedScene = preload("uid://stdqc6ttomff")
+const FRAYS_END := "uid://cufkthb25mpxy"
```

Comment:
At this point I wonder if Fray's End UID should be in a global/singleton/autoload.

Reply:
The line above this also explains something I couldn't understand when fixing #1216: even after I removed the `PackedScene` reference between the splash screen and the title screen, `Cinematic` was still getting loaded. I think it was by this!